playhand.js

```diff
@@ -532,7 +555,26 @@ function myPokerHand(handCards, boardCards) {
     });
 
     // Check straight flush
-    // will do this later
+    var flush = false
+    var straightFlush = false
```

Comment:
nitpick but wanna call these `hasFlush` and `hasStraightFlush` since they're booleans?